hack/.deployUtils

```diff
@@ -47,7 +47,7 @@ deployOCMHub(){
     clusterName=kind-${KIND_CLUSTER_CONTROL_PLANE}
     echo "Found single cluster installing hub and spoke on the one cluster (${clusterName})"
     join=$(${CLUSTERADM_BIN} get token --context ${clusterName} |  grep -o  'clusteradm.*--cluster-name')
-    ${join} ${clusterName} --bundle-version='0.11.0' --feature-gates=RawFeedbackJsonString=true --force-internal-endpoint-lookup --context ${clusterName} | grep clusteradm
+    ${BIN_DIR}/${join} ${clusterName} --bundle-version='0.11.0' --feature-gates=RawFeedbackJsonString=true --force-internal-endpoint-lookup --context ${clusterName} | grep clusteradm
```

Comment:
it should work. https://github.com/Kuadrant/multicluster-gateway-controller/pull/371/files#diff-dd9adc823d5cb403f5710f23e6624096dd323f422723e7672b0f76c51385d849R21 looks ugly to me, but can't think of a more elegant solution

Reply:
What is the BIN_DIR value?

Reply:
in case of the local setup it will be `./../bin` and the call will look like `<path to the hack folder>/../bin/clusteradm` and in case of the quicstartscript it will be an empty srting just calling `clusteradm`

Reply:
sorry it's not clear to me why we need to add this because that's already part of the declaration of join as CLUSTERADM_BIN. 🤔

Reply:
if we don't have it `local-setup` stops working (the script will be fine though). It is a bit funky and it evaluates join and  on the line 50 it starts as `clusteradm` even when we are having the `CLUSTERADM_BIN` pointing to the binary 
Here. This code 
```
test(){
  clusterName=${1}

  echo ${BIN_DIR}
  echo ${CLUSTERADM_BIN}
  join=$(${CLUSTERADM_BIN} get token --context kind-${clusterName} |  grep -o  'clusteradm.*--cluster-name')
  echo ${join}
  echo ${BIN_DIR}/${join}
}
```
prompts 
```
/home/hudeb/repos/multicluster-gateway-controller/hack/../bin
/home/hudeb/repos/multicluster-gateway-controller/hack/../bin/clusteradm
clusteradm join --hub-token ... --hub-apiserver https://127.0.0.1:39139 --cluster-name
/home/hudeb/repos/multicluster-gateway-controller/hack/../bin/clusteradm  join --hub-token ... --hub-apiserver https://127.0.0.1:39139 --cluster-name
```

Reply:
Strange. okay, so we need to set BIN_DIR to empty string in the quickstart script then yes?

Reply:
actually, it would keep `/` in front of it for the quickstart. But now it won't